src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.vb

```diff
@@ -433,6 +434,171 @@ End Class
+                    Dim method = bar.GetMember(Of MethodSymbol)("Method")
+                    Dim parameters = method.Parameters
+                    Dim theParameter = DirectCast(parameters(0), PEParameterSymbol)
+                    Dim peModuleSymbol = DirectCast(peModule, PEModuleSymbol)
+
+                    Assert.Equal(ParameterAttributes.None, theParameter.ParamFlags)
+
+                    ' let's find the attribute in the PE metadata
+                    Dim attributeInfo = CodeAnalysis.PEModule.FindTargetAttribute(peModuleSymbol.Module.MetadataReader, theParameter.Handle, AttributeDescription.DateTimeConstantAttribute)
+                    Assert.True(attributeInfo.HasValue)
+
+                    Dim attributeValue As Long
+                    Assert.True(peModuleSymbol.Module.TryExtractLongValueFromAttribute(attributeInfo.Handle, attributeValue))
+                    Assert.Equal(-1L, attributeValue)
+
+                    ' check .param has no value
+                    Dim constantHandle = peModuleSymbol.Module.MetadataReader.GetParameter(theParameter.Handle).GetDefaultValue()
+                    Assert.True(constantHandle.IsNil)
+                End Sub
+
+            CompileAndVerify(source, symbolValidator:=symValidator)
+        End Sub
+
+        <WorkItem(217740, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=217740")>
+        <Fact()>
+        Public Sub LoadingDateTimeConstantWithBadValueOnField()
+            Dim ilSource = <![CDATA[
+.class public auto ansi C
+       extends [mscorlib]System.Object
+{
+  .field private static initonly valuetype [mscorlib]System.DateTime f
+  .custom instance void [mscorlib]System.Runtime.CompilerServices.DateTimeConstantAttribute::.ctor(int64) = ( 01 00 ff ff ff ff ff ff ff ff 00 00 )
+  .method public specialname rtspecialname
+          instance void  .ctor() cil managed
+  {
+    // Code size       7 (0x7)
+    .maxstack  8
+    IL_0000:  ldarg.0
+    IL_0001:  call       instance void [mscorlib]System.Object::.ctor()
+    IL_0006:  ret
+  } // end of method C::.ctor
+
+  .method public instance void  Method() cil managed
+  {
+    // Code size       29 (0x1d)
+    .maxstack  2
+    .locals init (valuetype [mscorlib]System.DateTime V_0,
+             valuetype [mscorlib]System.DateTime V_1,
+             valuetype [mscorlib]System.DateTime V_2,
+             valuetype [mscorlib]System.DateTime V_3,
+             valuetype [mscorlib]System.DateTime V_4,
+             valuetype [mscorlib]System.DateTime V_5,
+             valuetype [mscorlib]System.DateTime V_6,
+             valuetype [mscorlib]System.DateTime V_7,
+             valuetype [mscorlib]System.DateTime V_8,
+             valuetype [mscorlib]System.DateTime V_9,
+             valuetype [mscorlib]System.DateTime V_10,
+             valuetype [mscorlib]System.DateTime V_11,
+             valuetype [mscorlib]System.DateTime V_12)
```

Comment:
Done. 
This was actually emitted by the native VB compiler (with lots of un-necessary locals). I verified that Roslyn only emits one.